support/scheduler-client/client_test.go

```diff
@@ -0,0 +1,425 @@
+//
+// Copyright (c) 2017 Tencent
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package scheduler
+
+import (
+	"encoding/json"
+	"github.com/edgexfoundry/edgex-go/core/domain/models"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+	"testing"
+)
+
+// Test common const
+const (
+	TestUnexpectedMsg                    = "unexpected result"
+	TestUnexpectedMsgFormatStr           = "unexpected result, active: '%s' but expected: '%s'"
+	TestUnexpectedMsgFormatStrForBoolVal = "unexpected result, active: '%t' but expected: '%t'"
+)
+
+// Test comm api path
+const (
+	TestExpectedScheduleApiPath      = "/api/v1/schedule"
+	TestExpectedScheduleEventApiPath = "/api/v1/scheduleevent"
+)
+
+// Test Schedule model const fields
+const (
+	TestScheduleName      = "midnight-1"
+	TestScheduleStart     = "20000101T000000"
+	TestScheduleEnd       = ""
+	TestScheduleFrequency = "P1D"
+	TestScheduleCron      = "This is a description"
+	TestScheduleRunOnce   = true
+	TestScheduleIdForTest = "testScheduleId"
+)
+
+// Test ScheduleEvent model const fields
+const (
+	TestScheduleEventName                = "pushed events"
+	TestScheduleEventParameters          = ""
+	TestScheduleEventService             = "notifications"
+	TestScheduleEventSchedule            = "testSchedule"
+	TestScheduleEventAddressableName     = "MQTT"
+	TestScheduleEventAddressableProtocol = "MQTT"
+	TestScheduleEventIdForTest           = "testScheduleEventId"
+)
+
+// Test method : SendSchedule
+func TestSchedulerClient_SendSchedule(t *testing.T) {
```

Comment:
Rename function tests from TestSchedulerClient_SendSchedule to something like TestSendSchedule